services/core-api/app/api/users/resources/user_resource.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from flask_restx import Resource
+from pytz import utc
+from sqlalchemy.exc import IntegrityError
+
+from app.api.users.models.user import User
+from app.api.users.response_models import USER_MODEL
+from app.api.utils.include.user_info import User as UserUtils
+from app.api.utils.resources_mixins import UserMixin
+from app.extensions import api
+
+
+class UserResource(Resource, UserMixin):
+    @api.doc(description='Retrive a list of alerts for a mine')
+    @api.marshal_with(USER_MODEL, code=200)
+    def get(self):
```

Comment:
Do you think we maybe should restrict this to a base role?
Thinking we might not want to create a user record if they actually don't have the role to view core

Reply:
Done!